pyop2/configuration.py

```diff
@@ -74,6 +90,12 @@ class Configuration(dict):
         cdim > 1 be built as block sparsities, or dof sparsities.  The
         former saves memory but changes which preconditioners are
         available for the resulting matrices.  (Default yes)
+    :param vectorization_strategy: A :class:`str` describing the
+        vectorization strategy that must to be applied to the kernels. Can
+        be one of the following --
+
+        - ``sun2020study``: Cross-element vectorization strategy of
```

Comment:
Would it not be better to call this `"cross-element"`? Also this sort of configuration can be good to put into an `enum`.

Reply:
Since there is only the choice between `"cross-element"` and `""` for the vectorisation strategy, an enum is a bit overkill?